### PR TITLE
DOC: add doc/source/contributing.rst as the CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jreback**
xref #9754 

I just did

`pandoc doc/source/contributing.rst -t markdown > CONTRIBUTING.md`

to generate the new CONTRIBUTING file.

### CODE REVIEW

**Comment by jreback** [comment]
cc @rockg
@jorisvandenbossche

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,105 +1,569 @@
...
+
+The code is hosted on [GitHub](https://www.github.com/pydata/pandas). To
+contribute you will need to sign up for a [free GitHub
+account](https://github.com/signup/free). We use
+[Git](http://git-scm.com/) for version control to allow many people to
+work together on the project.
+
+Some great resources for learning git:
+
+> -   the [GitHub help pages](http://help.github.com/).
```
Comment (shoyer) [inline CONTRIBUTING.md]: could remove the redundant blockquotes `>`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,105 +1,569 @@
...
+    origin  git@github.com:yourname/pandas.git (push)
+    upstream        git://github.com/pydata/pandas.git (fetch)
+    upstream        git://github.com/pydata/pandas.git (push)
+
+Now your code is on GitHub, but it is not yet a part of the *pandas*
+project. For that to happen, a Pull Request needs to be submitted on
+GitHub.
+
+Contributing your changes to *pandas*
+\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~\~
```
Comment (shoyer) [inline CONTRIBUTING.md]: This section title didn't get converted properly

**Comment by jreback** [comment]
ok, so this is the question. I can edit this file, but then these are disconnected.

anyone used pandoc for this before?

**Pull request closed without merging**